docs/how-to/create-beamline.rst

```diff
@@ -43,13 +42,27 @@ The following example creates a fictitious beamline ``w41``, with a simulated tw
     to one or more Bluesky Protocols.
     """
 
+    """
+    A valid factory function which:
+    - may be instantiated automatically, selectively on live beamline
+        - a maximum of once
```

Comment:
Should: At a first glance this implies you should only call `synchrotron()` once, not that it's cached

Reply:
How's the new wording, other than wordier?